google-cloud-spanner/src/main/java/com/google/cloud/spanner/testing/RemoteSpannerHelper.java

```diff
@@ -50,6 +50,8 @@ public class RemoteSpannerHelper {
   private final InstanceId instanceId;
   private static final AtomicInteger dbSeq = new AtomicInteger();
   private static final int dbPrefix = new Random().nextInt(Integer.MAX_VALUE);
+  private static int dbRoleSeq;
```

Comment:
Although the probability is low that it goes wrong, it would be better to use an `AtomicInteger` for this, as our integration tests can run in parallel.